modules/programs/firefox/mkFirefoxModule.nix

```
@@ -0,0 +1,984 @@
+modulePath:
+{ name, description ? null, wrappedPackageName ? null
```

I think these arguments could be merged?

```suggestion
{ modulePath, name, description ? null, wrappedPackageName ? null
```

---

I took inspiration from another mk method for this, which I cannot find currently. Not sure why the arguments were separated there, probably because it's not a property of the config, but rather of the module itself. The same attribute layout is used in the test factories. Changed it nonetheless.